examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonPartitionExample.scala

```diff
@@ -101,17 +126,40 @@ object CarbonPartitionExample {
     spark.sql("""
        | CREATE TABLE IF NOT EXISTS t5
        | (
+       | id Int,
        | vin String,
        | logdate Timestamp,
        | phonenumber Long,
-       | area String
+       | area String,
+       | salary Int
        |)
        | PARTITIONED BY (country String)
        | STORED BY 'carbondata'
        | TBLPROPERTIES('PARTITION_TYPE'='LIST',
-       | 'LIST_INFO'='(China,United States),UK ,japan,(Canada,Russia), South Korea ')
+       | 'LIST_INFO'='(China, US),UK ,Japan,(Canada,Russia, Good, NotGood), Korea ')
        """.stripMargin)
 
+    // load data into partition table
+    spark.sql(s"""
+       LOAD DATA LOCAL INPATH '$testData' into table t0 options('BAD_RECORDS_ACTION'='FORCE')
+       """)
+    spark.sql(s"""
+       LOAD DATA LOCAL INPATH '$testData' into table t5 options('BAD_RECORDS_ACTION'='FORCE')
+       """)
+
+    // alter list partition table t5 to add a partition
+    spark.sql(s"""Alter table t5 add partition ('OutSpace')""".stripMargin)
+    // alter list partition table t5 to split partition 4 into 3 independent partition
+    spark.sql(
+      s"""
+         Alter table t5 split partition(4) into ('Canada', 'Russia', '(Good, NotGood)')
+       """.stripMargin)
```

Comment:
after doing this, can you verify the partition is correct by using desc formatted?